pycommon/report2idea.py

```diff
@@ -111,7 +111,7 @@ def Run(module_name, module_desc, req_type, req_format, conv_func, arg_parser =
     arg_parser.add_argument('-v', '--verbose', action='store_true',
                             help="Enable verbose mode (may be used by some modules, common part donesn't print anything")
     arg_parser.add_argument('--whitelist-file', metavar="FILE", type=str,
-                            help="File with whitelist information in format:<ip address>/<mask>,<data>\\n \n <data> and <mask> are optional")
+                            help="File with whitelist filter information in format:<ip address>/<mask>,<data>\\n \n <data> and <mask> are optional. Whitelist is applied to SRC_IP field and report only ip address from whitelist file.")
```

Comment:
This whitelisting logic is wrong.
Addresses on whitelist shouldn't be reported, all others should be reported.